docs/REFERENCE.md

```diff
@@ -141,7 +141,6 @@ produce an explicit or implicit client response.
 | Attribute(s)            | Type             | Default | Description                                                                                                       | Characteristic(s)                                                                                                                                                                                                                                                                                                                                                                                                                               | Example                                                              |
 |:------------------------|:-----------------|:--------|:------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------|
 | `request_body_limit`    | string           | `"64MiB"` | Configures the maximum buffer size while accessing `request.form_body` or `request.json_body` content.          | &#9888; Valid units are: `KiB, MiB, GiB`                                                                                                                                                                                                                                                                                                                                                                                                        | `request_body_limit = "200KiB"`                                      |
-| `path`                  | string           | -       | Changeable part of the upstream URL. Changes the path suffix of the outgoing request.                             | -                                                                                                                                                                                                                                                                                                                                                                                                                                               | -                                                                    |
```

Comment:
:warning: We have to remove `path` in [intro text](#diff-e01aa12b8a02153953e3632db909ffac654ed44ce58a25e938b4f6d3208deb69L129-L133) and in [`docs/README.md`](README.md) as well!